tests/phpunit/CRM/Core/ComposerConfigTest.php

```diff
@@ -4,7 +4,7 @@
  * Class CRM_Core_ComposerConfigTest
  * @group headless
  */
-class CRM_Core_ComposerConfigTest extends \PHPUnit_Framework_TestCase {
+class CRM_Core_ComposerConfigTest extends \PHPUnit\Framework\TestCase {
```

Comment:
So specifically these are safe
https://engineering.facile.it/blog/eng/phpunit-upgrade-namespace/